plugins/modules/dcnm_network.py

```diff
@@ -1055,7 +1064,11 @@ def get_diff_merge(self, replace=False):
                         self.module.fail_json(msg="Unable to generate networkId for network: {} "
                                                   "under fabric: {}".format(want_c['networkName'], self.fabric))
 
-                    create_path = '/rest/top-down/fabrics/{}/networks'.format(self.fabric)
+                    if not self.is_ms_fabric:
+                        create_path = '/rest/top-down/fabrics/{}/networks'.format(self.fabric)
+                    else:
+                        create_path = '/rest/top-down/fabrics/{}/networks'.format(self.inventory_data['displayValues'][2])
```

Comment:
Are we certain that the list index will always be `[2]` in `self.inventory_data['displayValues']`?